octomap_server/src/octomap_server.cpp

```diff
@@ -309,7 +309,9 @@ OctomapServer::OctomapServer(const rclcpp::NodeOptions & node_options)
     std::make_shared<tf2_ros::TransformListener>(*tf2_buffer_);
 
   using std::chrono_literals::operator""s;
-  point_cloud_sub_.subscribe(this, "cloud_in");
+  rclcpp::SensorDataQoS qos_subscription;
+  // qos_subscription.keep_last(input_queue_size_);
+  point_cloud_sub_.subscribe(this, "cloud_in", qos_subscription.get_rmw_qos_profile());
```

Comment:
@wxmerkt You can use `rmw_qos_profile_sensor_data` directly if you won't change the queue size.
https://github.com/ros2/rmw/blob/9f3c3e1b010a95013eb119ce2ae9b225bdfdc1ba/rmw/include/rmw/qos_profiles.h#L25

Reply:
Thank you for the hint! I think the PR is now good to go if you are happy with my changes.